packages/flutter_tools/lib/src/cache.dart

```diff
@@ -105,21 +107,38 @@ class Cache {
     }
   }
 
-  static String _dartSdkVersion;
+  String _dartSdkVersion;
 
-  static String get dartSdkVersion => _dartSdkVersion ??= platform.version;
+  String get dartSdkVersion => _dartSdkVersion ??= platform.version;
 
-  static String _engineRevision;
+  String _engineRevision;
 
-  static String get engineRevision {
```

Comment:
Cleaning up these static calls. As far as I can see, there's no need for them to be static. Then we can reuse the getVersionFor fn.